pwnagotchi/ui/view.py

```diff
@@ -23,7 +23,21 @@
 
 class View(object):
     def __init__(self, config, impl, state=None):
-        global ROOT
+        global ROOT, BLACK, WHITE
+        
+        self.invert = 0
+        self._black = 0xFF
+        self._white = 0x00
+        if 'invert' in config['ui'] and config['ui']['invert'] == True:
+            logging.debug("INVERT BLACK/WHITES:" + config['ui']['invert'])
+            self.invert = 1
+            BLACK = 0x00
+            WHITE - 0xFF
```

Comment:
Is this done on purpose or should it an assign like the one above?

Reply:
That WHITE is correct because it is inverted from the globals default for this image above are already "inverted" from other images, So for this image inverted means being white, and not inverted is black. Where for other images, they may be swapped because white is the default

Reply:
So is it correct to say `WHITE - 0xFF` rather than `WHITE = 0xFF`?

Reply:
oh haha, no its meant to be `=` but `-` works because `0x00` - `0xFF` = `-255` which displays think as white. will get it changed since some displays may not like it

Reply:
Stuff like this happen, this is why code reviews are useful! :)
Thanks for the feature, I love it!